src/public/less/styles.less

```diff
@@ -1,38 +1,39 @@
-
 /* ------------------------
 Selected Bootstrap Imports
 --------------------------*/
+// path to bootstrap source files
+@BOOTSTRAP: "../../../node_modules/bootstrap/less";
```

Comment:
set a var for the path to bootstrap in node_modules, and then use that var for all the imports below.